[14.0][ADD] l10n_pt_sale_invoice_spms

eantones opened the pull request.

deeniiz committed:
[ADD] l10n_pt_sale_invoice_spms

deeniiz committed:
[ADD] l10n_pt_sale_invoice_spms

dreispt commented:
@eantones @etobella Podemos haver merge y instalar en el servidor de TEST por favor?

eantones commented:
@dreispt This PR has been installed and ready for testing in test5 since the beginning of August.
<img width="1623" height="427" alt="image" src="https://github.com/user-attachments/assets/019cfd80-fb58-4a21-af5d-89e510ef48fc" />
We use Git-Aggregator, so even though it’s not merged in GitHub (the branch is still in development), it’s already included in the test environment via the repos.yaml file.
Everything has been ready in the test environment since then.
Please let me know if I can help you with anything.

eantones committed:
[IMP] l10n_pt_sale_invoice_spms: pre-commit auto fixes

oca-fix-manifest-website and oca-gen-addon-readme regenerated the manifest
website key and the README/index.html with the correct
odoo-nonstandalone-addons links (the repo-name fix landed via the copier
template update in the previous PR).

eantones committed:
[FIX] l10n_pt_sale_invoice_spms: use the auto_install manifest key

The manifest declared "autoinstall", which Odoo silently ignores, so this
glue module never auto-installed when both of its dependencies were present.
Use the valid auto_install key.

eantones committed:
[REF] l10n_pt_sale_invoice_spms: modernize super() and drop dead import

Use the argument-less super() form in _prepare_invoice_line and remove the
dead commented-out account_move_line import in models/__init__.py.
